bin/nyc.js

```diff
@@ -46,6 +46,11 @@ var yargs = require('yargs/yargs')(process.argv.slice(2))
     type: 'boolean',
     describe: 'whether or not to instrument all files of the project (not just the ones touched by your test suite)'
   })
+  .option('include', {
+    alias: 'in',
+    default: [],
```

Comment:
The code arrives at that `**` default implicitly. I think we'd need a guard [here](https://github.com/bcoe/nyc/blob/deee980b965cbe70b2da2c70b28ce9fc0b4a65d5/index.js#L40) for empty `config.include` arrays.